bundles/subclipse.ui/src/org/tigris/subversion/subclipse/ui/wizards/CloudForgeComposite.java

```diff
@@ -24,26 +24,29 @@ public CloudForgeComposite(Composite parent, int style) {
 	}
 	
 	private void createControls() {	
-		Composite cloudForgeComposite = new Composite(this, SWT.NULL);
-		GridLayout layout = new GridLayout();
-		layout.numColumns = 1;
-		cloudForgeComposite.setLayout(layout);
-		GridData data = new GridData(GridData.GRAB_HORIZONTAL | GridData.FILL_HORIZONTAL);
-		cloudForgeComposite.setLayoutData(data);
+		String bannerDisabled = System.getProperty("disable.banner");
```

Comment:
Just a quick initial comment.  If we go down this path of a system property I would suggest we name it something like "subclipse.disableBanner"